cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -84,7 +84,12 @@ type Etcd struct {
 type NodeConfiguration struct {
 	metav1.TypeMeta
 
-	Discovery Discovery
+	DiscoveryURL      string
+	DiscoveryFile     string
+	DiscoveryToken    string
+	TLSBootstrapToken string
+	Token             string
+	Masters           []string
```

Comment:
Currently we only pay attention to one api server but hope to support >1 in the future.  That should be documented.